content/docs/command-reference/run.md

```diff
@@ -6,13 +6,16 @@ command.
 ## Synopsis
 
 ```usage
-usage: dvc run [-h] [-q | -v] [-n <name>] [--no-exec] [--no-run-cache]
-               [--no-commit] [-d <path>] [-o <filename>] [-O <filename>]
-               [-p [<filename>:]<params_list>] [-m <path>] [-M <path>]
-               [--plots <path>] [--plots-no-cache <path>] [--live <path>]
-               [--live-no-cache <path>] [--live-no-html]
-               [-w <path>] [-f] [--outs-persist <filename>]
-               [--outs-persist-no-cache <filename>] [-c <filename>]
+usage: dvc run [-h] [-q | -v] [-n <name>]
+               [--no-exec] [--no-run-cache] [--no-commit]
+               [-d <path>] [-p [<filename>:]<params_list>]
+               [-o <filename>] [-O <filename>] [-c <filename>]
+               [--outs-persist <filename>]
+               [--outs-persist-no-cache <filename>]
+               [-m <path>] [-M <path>]
+               [--plots <path>] [--plots-no-cache <path>]
+               [--live <path>] [--live-no-cache <path>]
+               [--live-no-html] [-w <path>] [-f]
                [--always-changed] [--external] [--desc <text>]
                command
```

Comment:
OR: Here's a new way to organize the usage block. Its sorted manually by related flags (does not preserve the semi-random order from the help output) + it's all under 68 chars long -- no scroll bar even on Mac:

![image](https://user-images.githubusercontent.com/1477535/157405352-0486cf0d-6d9d-4c4a-85fe-a323cda375a2.png)

Reply:
I think this version is better.

Reply:
Agree but we'd probably want the tool itself (core repo) to use the same order. WDYT @dberenbaum ?

Reply:
I'm not sure if the core uses [custom formatter](https://docs.python.org/3/library/argparse.html#formatter-class), but from the looks of it, it's not an easy feat to specify the sorting of these. I think we can have slightly different sorting between the docs and the tool.

Reply:
From what I remember it's just the order in which args are added in the code itself so it has to be done manually. I'd be happy to contribute the PR to match this one but after that we'd need a little project (issue) to check all the commands.